src/js/components/CompleteYourProfile/CompleteYourProfile2024.jsx

```diff
@@ -365,7 +365,7 @@ const HowItWorksContainer = styled('div')`
   alignItems: 'center';
   // height: 168px;
   border-radius: 10px;
-  border: 1px solid ${Colors.lightGrey};
+  border: 1px solid #B0B0B0;
```

Comment:
I really like using a Colors object like you did before instead of putting a color directly in the component. Can we go back to that, but choose another standardized color?